sycl/doc/extensions/experimental/sycl_ext_intel_fpga_task_sequence.asciidoc

```diff
@@ -0,0 +1,463 @@
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+:dpcpp: pass:[DPC++]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+== Notice
+
+[%hardbreaks]
+Copyright (C) 2022-2023 Intel Corporation.  All rights reserved.
+
+Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks
+of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by
+permission by Khronos.
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/intel/llvm/issues
+
+== Contributors
+
+// spell-checker: disable
+Jessica Davies, Intel +
+Joe Garvey, Intel +
+Robert Ho, Intel +
+Tommy Hoffner, Intel +
+Ajaykumar Kannan, Intel +
+Michael Kinsner, Intel +
+Abhishek Tiwari, Intel
+// spell-checker: enable
+
+== Dependencies
+
+This extension is written against the SYCL 2020 revision 6 specification.  All
+references below to the "core SYCL specification" or to section numbers in the
+SYCL specification refer to that revision.
+
+This extension also depends on the following other SYCL extension:
+
+* link:./sycl_ext_oneapi_properties.asciidoc[sycl_ext_oneapi_properties]
+* link:./sycl_ext_oneapi_kernel_properties.asciidoc[
+  sycl_ext_oneapi_kernel_properties]
+
+== Status
+
+This is an experimental extension specification, intended to provide early
+access to features and gather community feedback.  Interfaces defined in this
+specification are implemented in {dpcpp}, but they are not finalized and may
+change incompatibly in future versions of {dpcpp} without prior notice.
+*Shipping software products should not rely on APIs defined in this
+specification.*
+
+== Backend support status
+
+This extension is currently implemented in `icpx` only for FPGA devices and
+only when using either the Level Zero backend or the Intel FPGA OpenCL backend.
+Attempting to use this extension in kernels that run on other devices or
+backends may result in undefined behavior.  Be aware that the compiler is not
+able to issue a diagnostic to warn you if this happens.
+
+== Overview
+
+Although SYCL supports task-level parallelism through device kernels, there is
+no specific mechanism to express parallelism at a sub-kernel level or to invoke
+additional kernels without involving the host program. In OpenCL 2.0 and beyond,
+kernel invocations can be initiated without host program interaction through
+the `enqueue_kernel` API. C++11 also introduced `std::async()` for task-level
+parallelism. However, neither of these approaches is a good fit for spatial
+compute platforms, which require a clear mapping of parallel constructs to
+spatial instances in the hardware.
+
+Rather than significantly changing the semantics of existing solutions, this
+extension introduces a sub-kernel task-level parallelism interface to SYCL for
+use in device code. It provides the class
```

Comment:
To make it more readable, can you change the last sentence to:
> It provides the class sycl::ext::intel::experimental::task_sequence, which is an abstraction of an asynchronous sequence of invocations of a callable that can be executed asynchronously from the caller.  A single object of this class can map naturally to a single hardware instance on spatial targets.

Reply:
Updated.